BLOG_TOPIC_SUGGESTIONS.md

```diff
@@ -1,3 +1,7 @@
 This is a list of ideas of what we would like to publish.
 
-- CocoaPods vs Carthage vs Swift Package Manager (SPM)
+- CocoaPods vs Carthage vs Swift Package Manager (SPM).
+- Dealing with names.
+- Dealing with languages and localisation.
+- Stemming, lemmatisation and word separation.
```

Comment:
I have no idea of what this means, sounds interesting, it would be awesome if you could expand a bit. :D